src/server/connection.rs

```diff
@@ -1491,8 +1493,55 @@ impl Connection {
         self.video_ack_required = lr.video_ack_required;
     }
 
+    #[cfg(target_os = "windows")]
+    async fn handle_multiple_user_sessions(&mut self, lr: LoginRequest) -> bool {
+        if self.port_forward_socket.is_some() {
+            return true;
```

Comment:
`self.port_forward_socket` is always none here, you should judge with `lr.union`